glean-core/rlb/Cargo.toml

```diff
@@ -33,6 +33,7 @@ thiserror = "1.0.4"
 serde_json = "1.0.44"
 serde = { version = "1.0.104", features = ["derive"] }
 uuid = { version = "0.8.1", features = ["v4"] }
+time = "0.1.40" # Match version from mozilla-central
```

Comment:
Do we need the comment here? m-c needs to match basically with everything here, right?

Reply:
```suggestion
time = "0.1.40"
```